### PR TITLE
Enable GitHub Actions

### PR DESCRIPTION
**Pull request by yahonda**
This pull request enables GitHub Actions to run unit tests.

- Use ubuntu-20.04
- Use Ruby 2.7
- CI against SQLite3, MySQL and PostgreSQL databases
- CI against Rails v6.1.0.rc1, v6.0.3, 6-0-stable, 5-2-stable and v5.2.4
- Add MYSQL_USERNAME and MYSQL_PASSWORD environment variables

Refer https://github.com/activerecord-hackery/ransack/pull/1176#issuecomment-735644447 for the background of this pull request.

### CODE REVIEW

**Review by deivid-rodriguez — CHANGES_REQUESTED** [review]
Thanks @yahonda, this is awesome! :tada: 

Could you also replace the Travis badge in the README with Github Actions, and replace any other mentions to TravisCI in the repo?

Also, it's great to verify that ransack works fine with ruby 2.7, but I guess we should keep testing against 2.6.6 just in case. Could add a matrix.ruby dimension and test both?

Finally, and as a little nitpick, could you use an exact version, like 2.7.2. It's a personal preference, and it's what was previously used in the TravisCI configuration, so we should keep it that way.

**Comment by deivid-rodriguez** [comment]
And, and we should also completely delete the previous travis configuration :)

**Comment by yahonda** [comment]
Thanks for the review. Sure let me update this pull request as suggested.

File: README.md [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 Please see the [Maintainer wanted issue](https://github.com/activerecord-hackery/ransack/issues/1159) if you are interested.
 
-[![Build Status](https://travis-ci.org/activerecord-hackery/ransack.svg)](https://travis-ci.org/activerecord-hackery/ransack)
+[![Build Status](https://github.com/activerecord-hackery/ransack/workflows/test/badge.svg)](https://github.com/activerecord-hackery/ransack/actions)
```
Comment (deivid-rodriguez) [inline README.md:7]: I think it's `https://github.com/activerecord-hackery/ransack/workflows/Test/badge.svg`. With lowercase I get "Not found".

Comment (deivid-rodriguez) [reply README.md:7]: Either that or change the workflow name to "test".

Comment (yahonda) [reply README.md:7]: Right. I prefer to use the lowercase "test".

**Review by deivid-rodriguez — APPROVED** [review]

**Comment by deivid-rodriguez** [comment]
Merging to get a CI back on track!

**Comment by deivid-rodriguez** [comment]
Thanks so much @yahonda!

**Pull request merged**